.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-**Please refer to the contribution guidelines in the README when submitting PRs.**
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
```

Comment:
We should be aware that `Fixes #xxxx` automatically closes issues when merging PRs. Perhaps this is the desired behavior, so I'm in favor of it (if the rest of the team agrees). 👍 (Alternatively, we could use `Addresses #xxxx`)